src/library/Uno.Themes.WinUI.Markup/Theme/DatePicker.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class DatePicker
+		{
+			public static class Resources
+			{
+				public static class FlyoutButton
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerFlyoutButtonBackground")]
+						public static ResourceValue<Brush> Default => new("DatePickerFlyoutButtonBackground", true);
+					}
+				}
+
+				public static class FlyoutPresenter
+				{
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerFlyoutPresenterBackground")]
+						public static ResourceValue<Brush> Default => new("DatePickerFlyoutPresenterBackground", true);
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerFlyoutPresenterBorderBrush")]
+						public static ResourceValue<Brush> Default => new("DatePickerFlyoutPresenterBorderBrush", true);
+					}
+
+					public static class SpacerFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerFlyoutPresenterSpacerFill")]
+						public static ResourceValue<Brush> Default => new("DatePickerFlyoutPresenterSpacerFill", true);
+					}
+
+					public static class HighlightFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerFlyoutPresenterHighlightFill")]
+						public static ResourceValue<Brush> Default => new("DatePickerFlyoutPresenterHighlightFill", true);
+					}
+
+					public static class ContainerCornerRadius
+					{
+						[ResourceKeyDefinition(typeof(CornerRadius), "DatePickerFlyoutPresenterCornerRadius")]
+						public static ResourceValue<CornerRadius> Default => new("DatePickerFlyoutPresenterCornerRadius", true);
+					}
```

Comment:
Since these don't have visual state variants, we can make them direct properties of the FlyoutPresenter class instead of having a `Default` property each